static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -343,6 +345,10 @@ define([
             displayed = true;
         },
         addSlot = function ($adSlot) {
+            if (!commercialFeatures.dfpAdvertising) {
+                throw new Error('DFP advertising is disabled');
```

Comment:
I'm thinking that if there's a specific error we can anticipate, throwing a custom message with the relevant reason makes diagnosis easier.